packages/ui/src/services/icon-config/types.ts

```diff
@@ -107,4 +107,6 @@ export type IconConfig = {
   spin?: 'clockwise' | 'counter-clockwise'
 }
 
-export type IconsConfig = IconConfig[]
+type IconsConfig = IconConfig[]
+
+export type { IconsConfig, IconConfig }
```

Comment:
Any reason to export default as opposed to export everything?
I mean all types are being used, no?